torch/csrc/generic/StorageMethods.cpp

```diff
@@ -36,7 +36,11 @@ static PyObject * THPStorage_(isPinned)(THPStorage *self)
     cudaGetLastError();
     Py_RETURN_FALSE;
   }
-  return PyBool_FromLong(attr.memoryType == cudaMemoryTypeHost);
+  #if CUDA_VERSION >= 10000
+    return PyBool_FromLong(attr.type == cudaMemoryTypeHost);
```

Comment:
Huh, I'm pretty sure we're building with CUDA 10. Why haven't we caught this before?

Reply:
We do catch it. Search for "StorageMethods.cpp" here: https://circleci.com/api/v1.1/project/github/pytorch/pytorch/474693/output/104/0?file=true